e2xgrader/apps/api.py

```diff
@@ -140,3 +144,187 @@ def get_task_submissions(self, assignment_id, notebook_id, task_id):
             submission['index'] = idx
 
         return submissions
+
+class E2X_Gradebook(Gradebook):
+
+    def update_grade(self, grade_cell: str, notebook: str, assignment: str, student: str, **kwargs: dict) -> None:
+        """Updates grade for a submissions for a particular cell in a particular notebook in an assignment.
+
+        Arguments
+        ---------
+        grade_cell: string
+            The name of the grade cell
+        notebook_id: string
+            The name of the notebook
+        assignment_id: string
+            The name of the assignment
+        student: string
+            The id of the student
+        Returns
+        -------
+        None
+        """
+        try:
+            grade = self.find_grade(grade_cell, notebook, assignment, student)
+        except MissingEntry:
+            self.logger.warning('No grade found!')
```

Comment:
Shouldn't this be self.log.warning?

Reply:
This function was not required and hence deleted.